src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,115 @@
 import React from 'react';
 import './TodoList.scss';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.Component {
+  state = {
+    title: '',
+    selectedTodos: 'All',
+  };
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
+  }
+
+  render() {
+    const { todos, changeCheked, selectedUserId } = this.props;
+    const { title, selectedTodos } = this.state;
+
+    const newTodos = todos.filter(item => (
+      item.title.toLocaleLowerCase().includes(title.toLocaleLowerCase())
+    )).filter((item) => {
+      switch (selectedTodos) {
+        case 'Active':
+          return !item.completed;
+        case 'Completed':
+          return item.completed;
+        default:
+          return item;
+      }
+    });
```

Comment:
filter todos in App (before passing to Todolist)

Reply:
also move the functions inside filter methods to the separate func, so you can use them like this
`todos.filter(this.filterByQuery).filter(this.filterByStatus)`